src/components/interfaces/MOBILE_API.xml

```diff
@@ -5283,19 +5375,12 @@
   </function>
 
   <function name="OnHashChange" functionID="OnHashChangeID" messagetype="notification">
-	<description>
-		Notification containing an updated hashID which can be used over connection cycles (i.e. loss of connection, ignition cycles, etc.).
-		Sent after initial registration and subsequently after any change in the calculated hash of all persisted app data.
-	</description>
-	<param name="hashID" type="String" maxlength="100" mandatory="true">
-		<description>Calculated hash ID to be referenced during RegisterAppInterface.</description>
-	</param>
-  </function>
-
-  <function name="OnWayPointChange" functionID="OnWayPointChangeID" messagetype="notification">

```

Comment:
@LuxoftAKutsan reverted in https://github.com/LuxoftAKutsan/sdl_core/pull/113/commits/ab9adc7c6996e68e910941d6e3ed04f751e49215